packages/expo-media-library/ios/EXMediaLibrary/EXMediaLibrary.m

```diff
@@ -24,6 +24,8 @@
 
 NSString *const EXMediaLibraryDidChangeEvent = @"mediaLibraryDidChange";
 
+NSString *const EXMediaLibraryCachesDirectory = @"MediaLibrary";
```

Comment:
I'm not sure if this variable name should be prefixed with `EX`. It won't break anything. However, I don't think we want to have a separate cache folder for each SDK version. cc @tsapeta

Reply:
@tsapeta could you provide some input?

Reply:
Sorry for slow reaction 😅 It's fine as is – FileSystem's cacheDirectory should already be versioned so I think we don't need to do this again here.
(For clarification, references prefixes by `EX` are being versioned by our tools when we add new SDKs to the project. In other words, `EXMediaLibrary` would be transformed to `ABI38_0_0EXMediaLibrary`, but `MediaLibrary` would remain unchanged.)